test/test_core.py

```diff
@@ -7721,16 +7704,12 @@ def test_embind_dynamic_initialization(self):
 
   @no_wasm2js('wasm_bigint')
   def test_embind_i64_val(self):
-    self.set_setting('WASM_BIGINT')
     self.emcc_args += ['-lembind']
-    self.node_args += shared.node_bigint_flags()
```

Comment:
Are there any remaining callers of `node_bigint_flags`?

Reply:
This one is not needed, but there are still two uses after it, in `emcc.py` and `gen_struct_info.py` which I am not sure about.

Reply:
To remove it in them, we might need to add node version checks...